### ISSUE TITLE
EventRecord + DoubleMap conversion on metadata fails

### ISSUE THREAD

**Issue by xlc** [polkadot-js/api#900]
```
VM104:37 2019-05-14 14:11:18   API/DECORATOR: FATAL: Unable to initialize the API:  Cannot convert 'DoubleMapType' via asType
window.console.error @ VM104:37
m @ logger.js:83
error @ logger.js:129
(anonymous) @ Base.js:379
async function (async)
(anonymous) @ Base.js:364
u.emit @ index.js:201
emit @ Provider.js:343
onSocketOpen @ Provider.js:159
```

Relates to https://github.com/paritytech/substrate/pull/2491 as it introduce usage of double map in substrate
Should be fixed by https://github.com/polkadot-js/api/issues/776

**Comment by jacogr** [comment]
EventRecord has a new `topics` field added - https://github.com/polkadot-js/api/blob/master/packages/types/src/type/EventRecord.ts#L81

https://github.com/paritytech/substrate/blob/master/srml/system/src/lib.rs#L344

**Comment by jacogr** [comment]
Closed in #901 & #902
